collect/collect.go

```diff
@@ -434,9 +429,7 @@ func (i *InMemCollector) Stop() error {
 		traces := i.Cache.GetAll()
 		for _, trace := range traces {
 			if trace != nil {
-				if !trace.GetSent() {
```

Comment:
is `GetSent` redundant/can be removed now?

Reply:
It was always redundant. Removed.